hw/ip/entropy_src/data/entropy_src.hjson

```diff
@@ -134,6 +148,24 @@
         }
       ]
     },
+    { name: "MODULE_ENABLE",
+      desc: "Module enable register",
+      swaccess: "rw",
+      hwaccess: "hro",
+      regwen:   "REGWEN_ME",
+      tags: [// Exclude from writes to these field because they cause side affects.
+             "excl:CsrAllTests:CsrExclAll"]
+      fields: [
+        { bits: "3:0",
+          mubi: true,
+          desc: '''
+                Setting this field to kMuBi4True will enable the ENTROPY_SRC module. Setting
+                this field to kMuBi4False will effectively reset the moduel.
```

Comment:
typo: moduel->module
What happens if !kMuBi4False and !kMuBi4True?
Same Q for other MuBis used by this module?

Reply:
If any mubi-based field is not equal to one of the two valid values, then a recov_alert is signaled, and a status bit is set. This is true for all entropy complex blocks.

Reply:
Thanks. SGTM. I assume you don't pass it through different clock domains and if you do, you take care of full synchronization. Right?

Reply:
There are no CDCs inside the entropy_src.  Any domain-crossing between the CPU and the IP is fully managed by TL fabric.